addons/dataverse/static/dataverseNodeConfig.js

```diff
@@ -253,7 +253,7 @@ ViewModel.prototype.updateFromData = function(data) {
         self.savedDataverseTitle(data.savedDataverse.title);
         self.selectedDataverseAlias(data.savedDataverse.alias);
         self.savedDatasetDoi(data.savedDataset.doi);
-        self.savedDatasetTitle(data.savedDataset.title);
+        self.savedDatasetTitle(data.savedDataset.name);
```

Comment:
If you connect a dataverse, and then connect a dataset and save, then refresh the page, I don't know if savedDatasetTitle is null or what, but the dataset title is not showing up.  If you connect one it shows, but not on first load.

![screen shot 2018-04-25 at 10 26 14 am](https://user-images.githubusercontent.com/9755598/39255919-3ec3cccc-4873-11e8-864f-80ab0b96b873.png)